elsevier/_skip.txt

```diff
@@ -16,9 +15,29 @@ aasri-procedia
 journal-of-adolescent-health
 materials-discovery-today
 pain
-cell
 journal-of-investigative-dermatology
 medical-journal-of-australia
 
 # Moved to Springer
 journal-of-ultrasound
+
+#Cell Press journals
+biophysical-journal
+cancer-cell
+cell
+cell-chemical-biology
+cell-host-and-microbe
+cell-metabolism
+cell-reports
+cell-stem-cell
+current-biology
+developmental-cell
+immunity
+molecular-cell
+neuron
+stem-cell-reports
+structure
+#Cell but different style
+the-american-journal-of-human-genetics
+#Cell, different style, not yet done
+molecular-plant
```

Comment:
I have this thread as my last note on Molecular plant: https://forums.zotero.org/discussion/57671/incorrect-style-for-moleculer-plant where I say

> The style is actually identical to https://www.zotero.org/styles?q=The%20Plant%20Cell